.gitignore

```diff
@@ -39,8 +39,5 @@ override.tf.json
 
 # Ignore Visual Studio Code config
 **/.vscode/*
+super-linter.log
 
-# Ignore Super-Linter log
```

Comment:
Please revert this to the upstream change as I added a couple of additional items to ignore for Super-Linter